Sources/VLCMovieViewController.m

```diff
@@ -1891,4 +1928,49 @@ - (void) mediaMoreOptionsActionSheetDidToggleInterfaceLockWithState:(BOOL)state
 {
     [self toggleUILock];
 }
+
+#pragma mark - VLCMediaPlaybackControlToolbar
+
+- (void)setupPlaybackControlToolbar
+{
+    _playbackControlToolbar = [[VLCMediaPlaybackControlToolbar alloc] init];
+    _playbackControlToolbar.delegate = self;
+    [self.view addSubview:_playbackControlToolbar];
+}
+
+#pragma mark - VLCMediaPlaybackControlToolbarDelegate
+
+- (void)handlePlayPauseNotification:(NSNotification *)notification
+{
+    // _vpc.isPlaying cannot be used here because it is nil since the notification is triggered before it is set
+    if (notification.name == VLCPlaybackControllerPlaybackDidResume ||
+        notification.name == VLCPlaybackControllerPlaybackDidPause) {
+        _playbackControlToolbar.isPlaying = notification.name == VLCPlaybackControllerPlaybackDidResume;
+    }
```

Comment:
that if is redundant I believe

Reply:
lol funnily enough I was afraid you'd say the line 1948 alone would be ambiguous so I added the if around it 😅